### PR TITLE
Add e2e workload test for multiple ctrs

### PR DESCRIPTION
**Pull request by machacekondra**
This commit add e2e test for multiple containers in workload,
with one container having same name as workload and second having
different name.

Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: test/e2e/e2e_test.go [hunk 1]
```diff
@@ -323,27 +349,41 @@ func edgeworkloadDeviceLabel(name string, labels map[string]string, hostport int
...
-	if secretRef != nil {
-		containers[0]["envFrom"] = append(envFrom, map[string]interface{}{"secretRef": map[string]string{"name": *secretRef}})
-	}
-	if configRef != nil {
-		containers[0]["envFrom"] = append(envFrom, map[string]interface{}{"configMapRef": map[string]string{"name": *configRef}})
+
+	containers := []map[string]interface{}{}
+	for i := 0; i < ctrCount; i++ {
+		// Let's use same name as workload for container, and different name
+		// if multiple contaiers, so we use both cases.
```
Comment (gciavarrini) [inline test/e2e/e2e_test.go:366]: s/contaiers/containers/

**Review by gciavarrini — APPROVED** [review]
lgtm

**Review by jakub-dzon — APPROVED** [review]

File: test/e2e/e2e_test.go [hunk 2]
```diff
@@ -77,6 +77,26 @@ var _ = Describe("e2e", func() {
...
+			_, err = workload.Create(edgeworkloadDeviceIdContainers("nginx", device.GetId(), hostPort, nginxPort, 2))
+			Expect(err).To(BeNil())
+
+			// then
+			// Check the edgedevice report proper state of workload:
+			err = device.WaitForWorkloadState("nginx", "Running")
+			Expect(err).To(BeNil(), "cannot get workload status for nginx workload")
+
+			// Check the nginx is serving content:
+			stdout, err := device.Exec(fmt.Sprintf("curl http://localhost:%d", hostPort))
```
Comment (eloycoto) [inline test/e2e/e2e_test.go:95]: Should we check here hostPort +1, no?

**Review by eloycoto — APPROVED** [review]

**Review by eloycoto — APPROVED** [review]

**Comment by eloycoto** [comment]
Thanks ondra, awesome work.

**Pull request merged**